microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+  CHECK_FAILURE(m_webView->get_DocumentTitle(&title));
+
+  // Passing nullptr for `ICoreWebView2PrintSettings` results in default print settings used.
+  CHECK_FAILURE(
+      webView2_16->PrintToPdfStream(
+          nullptr,
+          Callback<ICoreWebView2PrintToPdfStreamCompletedHandler>(
+              [title = std::move(title), this](HRESULT errorCode, IStream* pdfData) -> HRESULT
+              {
+                DisplayPdfDataInPrintDialog(pdfData);
+                std::wstring message = L"Printing " + std::wstring(title.get()) +
+                                       L" document to PDF Stream " +
+                                       ((errorCode == S_OK && pdfData != nullptr) ? L"succedded" : L"failed");
+
+                AsyncMessageBox(message, L"Print to PDF Stream");
+
+                return S_OK;
+              })
+              .Get()));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::PrintToPdfStream(printSettings, handler) method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#printtopdfstream)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## API Reference overview
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+* [CoreWebView2.ShowPrintUI() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui)
+* [CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui(microsoft-web-webview2-core-corewebview2printdialogkind))
+* [CoreWebView2.PrintAsync() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync#microsoft-web-webview2-core-corewebview2-printasync)
+* [CoreWebView2.PrintToPdfAsync(resultFilePath, printSettings) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfasync#microsoft-web-webview2-core-corewebview2-printtopdfasync)
+* [CoreWebView2.PrintToPdfStreamAsync(printSettings) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfstreamasync#microsoft-web-webview2-core-corewebview2-printtopdfstreamasync)
+* [CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+* [CoreWebView2PrintDialogKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printdialogkind)
+* [CoreWebView2PrintStatus Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printstatus)
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+* [ICoreWebView2_16::ShowPrintUI(printDialogKind) method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#showprintui)
+* [ICoreWebView2_16::Print() method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#print)
+* [ICoreWebView2_7::PrintToPdf(resultFilePath, printSettings, handler) method](/microsoft-edge/webview2/reference/win32/icorewebview2_7#printtopdf)
```

Comment:
```suggestion
* [ICoreWebView2_7::PrintToPdf method](/microsoft-edge/webview2/reference/win32/icorewebview2_7#printtopdf)
```
make this change if you decide to streamline link text for methods as we've recently done in other articles eg Overview of APIs

Reply:
Removed parenthesis and params wherever used in the doc except for ShowPrintUI method in print.md